### DISCUSSION TITLE
i dont have school email, but there is no place to provie more info

### DISCUSSION THREAD

**Discussion by Yggdrrasil** [community/community#143504] [GitHub Education]
### Hi everyone,

Hello, I want to get verified. When I try to, it says I need a school-issued email. I've read the forums and documentation, and it says I can use a certificate, which I have. However, I can't find a place or link to submit it 

on the documentation it says to go https://education.github.com/learner/learn

and then to "benefits," there is no "benefits,"  only " your benefits application." Also, if I click there, it takes me back to submit a school email I don't have. 

can any of you point me to the link to submit my application with my certificate and ID??

Thanks!

**Comment by misterob0t** [comment]
Hi @Yggdrrasil ,

Everything you are looking for is in this link : 

https://education.github.com/discount_requests/application

**Reply by Yggdrrasil** [reply]
Thanks for taking the time to reply but that link is not what I'm looking for 

that link only allows you to apply with student email, as I said In my post I don't have that, I need te link to upload a certificate and my id in order to prove I'm a student!
![image](https://github.com/user-attachments/assets/0b596791-6466-4c2d-bdc7-e70fd94d956a)

**Reply by misterob0t** [reply]
I used this link and I did not put my school email. There, you have to put the name of your school and then you put continue.

**Reply by Yggdrrasil** [reply]
This is what i get after entering my school name: 

![image](https://github.com/user-attachments/assets/5047a77b-dadd-4760-80b7-d292eabedf1e)

**Comment by MrTheMech** [comment]
same issue
my school has mails but they only give it to teachers and staff. students wont be given custom domain mails. they all use personal mails. but here they said it is must to add it in github...
